### ISSUE TITLE
What could be the reasons training loss is not low?

### ISSUE THREAD

**Issue by nyanmn** [weiliu89/caffe#916]
I have 1400 trained images and 350 test images. The issue is training loss is around 5 and doesn't get lower. But detection_eval is getting higher and higher. After 5000 iterations, detection evaluation hit to 82% but loss is still around 5. What could be the reason?
Training loss and learning rate are in the following figures.
[https://www.dropbox.com/s/ix92bcdxp2sjbsc/trainloss_iters.png?dl=0](url)
[https://www.dropbox.com/s/daahjke92p896n8/trainlearningrate_iters.png?dl=0](url)
